sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java

```diff
@@ -80,6 +81,209 @@ public EditCaseSteps(
                   "dateOfBirth"));
         });
 
+    When(
+        "I select Investigation Status Pending",
+        () ->
+            webDriverHelpers.clickWebElementByText(
+                INVESTIGATION_STATUS_OPTIONS,
+                CaseOutcome.getValueFor("INVESTIGATION PENDING").toUpperCase()));
+
+    When(
+        "I select Investigation Status Done",
+        () ->
+            webDriverHelpers.clickWebElementByText(
+                INVESTIGATION_STATUS_OPTIONS,
+                CaseOutcome.getValueFor("INVESTIGATION DONE").toUpperCase()));
+
+    When(
+        "I select Investigation Status Discarded",
+        () ->
+            webDriverHelpers.clickWebElementByText(
+                INVESTIGATION_STATUS_OPTIONS,
+                CaseOutcome.getValueFor("INVESTIGATION DISCARDED").toUpperCase()));
+
+    When(
+        "I check if date of investigation filed is available",
+        () -> webDriverHelpers.clickOnWebElementBySelector(INVESTIGATED_DATE_FIELD));
+
+    When(
+        "I select Outcome Of Case Status Deceased",
+        () ->
+            webDriverHelpers.clickWebElementByText(
+                OUTCOME_OF_CASE_OPTIONS, CaseOutcome.getValueFor("DECEASED").toUpperCase()));
+    When(
+        "I select Outcome Of Case Status Recovered",
+        () ->
+            webDriverHelpers.clickWebElementByText(
+                OUTCOME_OF_CASE_OPTIONS, CaseOutcome.getValueFor("RECOVERED").toUpperCase()));
+
+    When(
+        "I select Outcome Of Case Status Unknown",
+        () ->
+            webDriverHelpers.clickWebElementByText(
+                OUTCOME_OF_CASE_OPTIONS, CaseOutcome.getValueFor("UNKNOWN").toUpperCase()));
+
+    When(
+        "I check if date of outcome filed is available",
+        () -> webDriverHelpers.clickOnWebElementBySelector(DATE_OF_OUTCOME));
+
+    When(
+        "I click on yes option in Sequelae",
```

Comment:
I click on yes option in Sequelae  all these methods can be parametrized in BDD with Yes/No/Unknown as strings